templates/taxi/car_list.html

```diff
@@ -10,8 +10,11 @@ <h1>Car list</h1>
           {{ car.model }} ({{ car.manufacturer.name }})
         </li>
       {% endfor %}
+      <br>
+      <a class="btn btn-primary" href="{% url 'taxi:car-create' %}">add new car</a>
```

Comment:
Make sure to add a 'Cancel' button apart from the 'Delete' one. The 'Cancel' button should lead to the previous page the user was on, as per the checklist requirement.